kroxylicious-filters/kroxylicious-encryption/src/main/java/io/kroxylicious/filter/encryption/EnvelopeEncryption.java

```diff
@@ -105,7 +107,7 @@ private static <K, E> Kms<K, E> buildKms(FilterFactoryContext context, Config co
         kms = InstrumentedKms.wrap(kms, kmsMetrics);
         ExponentialJitterBackoffStrategy backoffStrategy = new ExponentialJitterBackoffStrategy(Duration.ofMillis(500), Duration.ofSeconds(5), 2d,
                 ThreadLocalRandom.current());
-        kms = ResilientKms.wrap(kms, context.eventLoop(), backoffStrategy, 3);
+        kms = ResilientKms.wrap(kms, RETRY_POOL, backoffStrategy, 3);
```

Comment:
We should add @NonNull to the formal parameters of ResilientKms.wrap to make intent clear.